knowledge-base/docker/setup-buildx-action/action-security.yml

```diff
@@ -0,0 +1,9 @@
+name: 'Docker Setup Buildx'
```

Comment:
@arjundashrath @h0x0er looks like this is a duplicate that you both analyzed. I already approved the one @h0x0er added. @arjundashrath while looking for endpoints, also do a search of the Action repo. Sometimes some outbound calls are conditional, so they are not made in each workflow run. As an example, in this case there is a call to `github.com`